dotnetv3/S3/ManageACLsExample/ManageACLs.cs

```diff
@@ -114,11 +120,11 @@ public static async Task<S3AccessControlList> GetBucketACLAsync(IAmazonS3 client
         }
 
         /// <summary>
-        /// Adds a new ACL to an existing object in the S3 bucket.
+        /// Adds a new ACL to an existing object in the Amazon S3 bucket.
         /// </summary>
         /// <param name="client">The initialized client object used to call
         /// PutBucketAsync.</param>
-        /// <param name="bucketName">A string representing the name of the S3
+        /// <param name="bucketName">A string representing the name of the Amazon S3
         /// bucket where the object we want to which we want to apply a new ACL.</param>
```

Comment:
Review this sentence for sense:
A string representing the name of the Amazon S3 bucket where the object we want to which we want to apply a new ACL

Reply:
Yeah, didn't make sense. Changed to:

A string representing the name of the Amazon S3 bucket containing the object to which we want to apply a new ACL.